lib/message.php

```diff
@@ -425,8 +426,12 @@ public function getListArray() {
 	/**
 	 * @return string
 	 */
-	public function getHtmlBody() {
-		return $this->htmlService->sanitizeHtmlMailBody($this->htmlMessage);
+	public function getHtmlBody($accountId, $folderId, $messageId, $attachments) {
```

Comment:
``$attachments`` is a closure, isn't it? We should type-hint it and add add info to the doc block. It's not clear how to use this method otherwise